spec/namespaces/snapshot.yaml

```diff
@@ -427,10 +427,11 @@ components:
           schema:
             type: object
             properties:
+              accepted:
+                description: Equals `true` if the restore was accepted. Present when the request had `wait_for_completion` set to `false`
```

Comment:
```suggestion
                description: Returns `true` if the restore was accepted. Present when the request had `wait_for_completion` set to `false`.
```

Reply:
The API actually returns a response in CamelCase.
```
{
  "test_index": {
    "shards": [
      {
        "id": 0,
        "type": "SNAPSHOT",
        "stage": "DONE",
        "primary": true,
        "start_time_in_millis": 1728835477686,
        "stop_time_in_millis": 1728835477725,
        "total_time_in_millis": 39,
        "source": {
          "repository": "test-repository",
          "snapshot": "6",
          "version": "2.17.0",
          "index": "test_index",
          "restoreUUID": "n7wQKoo_RpiFVXJB5BuKDQ",
          "isSearchableSnapshot": false,
          "remoteStoreIndexShallowCopy": false,
          "sourceRemoteStoreRepository": null,
          "sourceRemoteTranslogRepository": null
        },
        "target": {
          "id": "UtghWlpCT3akPM5wbVJxXA",
          "host": "172.19.0.2",
          "transport_address": "172.19.0.2:9300",
          "ip": "172.19.0.2",
          "name": "jdk21"
        },
        "index": {
          "size": {
            "total_in_bytes": 208,
            "reused_in_bytes": 0,
            "recovered_in_bytes": 208,
            "percent": "100.0%"
          },
          "files": {
            "total": 1,
            "reused": 0,
            "recovered": 1,
            "percent": "100.0%"
          },
          "total_time_in_millis": 23,
          "source_throttle_time_in_millis": 0,
          "target_throttle_time_in_millis": 0
        },
        "translog": {
          "recovered": 0,
          "total": 0,
          "percent": "100.0%",
          "total_on_start": 0,
          "total_time_in_millis": 7
        },
        "verify_index": {
          "check_index_time_in_millis": 0,
          "total_time_in_millis": 0
        }
      }
    ]
  }
}
```

Source code is also implemented in CamelCase.
https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/cluster/routing/RecoverySource.java#L413-L421

Reply:
Yes, please.